pages/common/jdeps.md

```diff
@@ -0,0 +1,24 @@
+# jdeps
+
+> Java Class Dependency Analyzer.
```

Comment:
Why the `Title Case`?
```suggestion
> Java class dependency analyzer.
```

Reply:
I originally did it like that because the `java`, `javac`, `jar`, `jmap` and some other java related commands did the same for those pages. But if title case isn't good I can make that change

Reply:
I think the `Title Case` doesn't fit here

Reply:
> I think the `Title Case` doesn't fit here

Agreed

Reply:
I can apply the change here so this page is already altered and then open another PR to change the other java related pages to no longer use title case as well. Does that seem like a good plan?

Reply:
yep